website/search/elastic_search.py

```diff
@@ -378,6 +385,41 @@ def update_user(user, index=None):
 
     es.index(index=index, doc_type='user', body=user_doc, id=user._id, refresh=True)
 
+@requires_search
+def update_file(file_, index=None, delete=False):
+
+    index = index or INDEX
```

Comment:
This line can be removed if you just write `index=INDEX` as the default kwarg value in L389.